grammar.js

```diff
@@ -991,6 +1016,7 @@ module.exports = grammar({
         $.interpolated_string_expression,
         $.capture_pattern,
         $.tuple_pattern,
+        $.named_tuple_pattern,
         $.case_class_pattern,
```

Comment:
I think we also need a modified case_class_pattern with the named arguments: https://docs.scala-lang.org/sips/named-tuples.html#pattern-matching-with-named-fields-in-general

Reply:
Addressed in 690838e56c20c2ab1ccaa033a54cc6ee8a787164